index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <style>
+        body{
+            background-color: rgba(104, 133, 80, 0.849);
+            background-image: url("https://wallpaperaccess.com/full/2041777.jpg");
+            text-emphasis-color: red;
+        }
+        #title{
+            text-align: center;
+            font-size: 40px;
+        }
+        #survey-form{
+            text-align: center;
+        }
+        #description{
+            text-align: center;
+            font-size: 20px;
+        }
+    </style>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>microverse survey form</title>
+</head>
+<body>
+    <h1 id="title" style = "color: red"> Survey form </h1>
+    <p id="description" style = "color: red"> This is a survey form </p>
+    <section>
+    <form id="survey-form" action="sumbit">
+        <label for = "name" id = "name-label"> Enter Your Name</label>
+        <input type="text" id="name" placeholder = "Enter your name" style = "color: red" required> </input>
+        <br>
+        <label for = "email" id = "email-label"> Enter Your Email</label>
+        <input type="email" id="email" placeholder = "Enter your email" style = "color: red" required></input>
+        <br>
+        <label for = "number" id = "number-label"> Enter Your Number</label>
+        <input type="number" min = "0000000000" max = "9999999999" id="number"  style = "color: red" placeholder = "Enter Your Phone Number" required></input>
+        <br>
+        <br>
+      <select id = "dropdown">
```

Comment:
You can create a different section for each part of the form and use class instead of an id to style that section.

Ex- <section>
<select id="dropdown">
-----------------------
</section>